src/App.tsx

```diff
@@ -15,36 +19,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+
+function sorteGoods(goods: string[], sortedField:string, reversed:boolean): string[] {
```

Comment:
The function name `sorteGoods` seems to be a typo. It should be `sortGoods` to reflect its purpose accurately.